COORDINATION.md

```diff
@@ -52,7 +52,9 @@ Infrastructure devices acting as either a Host peripheral or as an independent e
 
 ### Assumptions
 
-The infrastructure devices must be ready on the PCIe bus(es) at enumeration time which allows the driver to address the race condition.
+The infrastructure devices must follow the PCI spec and be ready on the bus(es)
+at enumeration time, and/or hot plug devices in and out dynamically.
```

Comment:
why two lines ? by mistake or intentional ?

Reply:
Yes I added a line break so it wouldn't wrap beyond 80 characters.